hw/block/nvme.c

```diff
@@ -2915,6 +2915,8 @@ static void lnvm_exit(NvmeCtrl *n)
     if (ln->meta_auto_gen)
         free(ln->meta_fname);
     fclose(ln->metadata);
+
+    printf("Remove lnvm\n");
```

Comment:
What is this used for? Can it be communicated some other way?

Reply:
It is to verify that the device is removed when issuing device_del. The command in the monitor is async, so this is an easy way to know when we can kill the machine then testing recovery.

Reply:
What if there are multiple nvme instances, then it will be hard to know which one exited. How about polling using "info" command? Would that be better?